app/views/layouts/application.html.erb

```diff
@@ -6,11 +6,24 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= render partial: "shared/cdn_assets" %>
```

Comment:
Great job adding your CDN to a partial, this helps clean up the application layout file :)